runtime/src/bank.rs

```diff
@@ -6355,6 +6363,9 @@ pub struct TotalAccountsStats {
     pub num_executable_accounts: usize,
     /// Total data size of executable accounts
     pub executable_data_len: usize,
+
+    /// Total number of rent exempt accounts
+    pub num_rent_exempt: usize,
```

Comment:
Request: I'm adding a new subcommand to ledger-tool that prints out this `TotalAccountsStats` at the end with `{:#?}` (see PR below), which then uses the field names directly. Would you be able to change this to `num_rent_exempt_accounts`?

Here's an example of the output:
```
TotalAccountsStats {
    num_accounts: 3716396,
    data_len: 91398670325,
    num_executable_accounts: 2674,
    executable_data_len: 131024153,
}
```

https://github.com/solana-labs/solana/pull/21163